Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -167,7 +167,7 @@ def calculate_norm(self, **kwargs):
         return s1
 
     @staticmethod
-    def dot_test(operator, domain_init=None, range_init=None, verbose=False, **kwargs):
+    def dot_test(operator, domain_init=None, range_init=None, verbose=False, tol=1e-6, **kwargs):

```

Comment:
I think these changes should be removed as already fixed by #1048

Reply:
Yes, already merged